3001-4000/LIT3082RepCh362.xml

```diff
@@ -73,9 +66,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <ptr target="bm:Dillmann1866Chrestomathia"/>
                   <citedRange unit="page">136-146</citedRange>
                </bibl>
+
+               <bibl>
+                  <ptr target="bm:MalkaGubae1974"/>
+                  <citedRange unit="page">112-122</citedRange>
+               </bibl>
```

Comment:
Denis was wishing a longer incipit eventually (checking also the book attached I guess, esp. in case it is any different). I will pre-approve but not merge, you can see if you merge or first add and then merge.

Reply:
Thank you for the reminder! I've expanded the first incipit according to Dillmann's text.